src/Innovator.Client/Command.cs

```diff
@@ -326,7 +326,7 @@ protected virtual void AddAml(string query)
     /// Create a command from an interpolated string
     /// </summary>
     /// <param name="formatted">Interpolated string to convert to a command</param>
-    public Command(FormattableString formatted)
+    public Command(FormattableString formatted) : this()
```

Comment:
![dailys-receipt](https://user-images.githubusercontent.com/35051621/34507816-cb245ade-f007-11e7-98bc-4bbfc0385eb3.jpg)
Filter issues and pull requests that involve, but don't necessarily @mention, certain people: state:open type:issue involves:octocat